### PR TITLE
Adding sign-up and scroll to Counterspell Minneapolis page

### PR DESCRIPTION
**Pull request by sophia-pung**
Add Sign-Up Button and Scrolling Feature to Minneapolis Page

### CODE REVIEW

**Comment by AverseABFun** [comment]
Just to be clear, this is approved my me, the organizer.

File: 0001-adding-a-sign-up-button-and-scrolling-to-Counterspel.patch
Comment (samdev-7) [inline 0001-adding-a-sign-up-button-and-scrolling-to-Counterspel.patch:1]: Not sure why this file is here, could you remove it?

File: package.json
Comment (samdev-7) [inline package.json:1]: Please don't modify this file. Revert your changes, thanks!

File: package-lock.json
Comment (samdev-7) [inline package-lock.json:1]: We use yarn `yarn install` & `yarn dev`. Please revert your changes on this file, thanks!

File: yarn.lock
Comment (samdev-7) [inline yarn.lock:1]: Please don't modify this file. Revert your changes, thanks!

**Review by samdev-7 — CHANGES_REQUESTED** [review]
Some quick changes and it should be good to go!

**Comment by samdev-7** [comment]
Closing this due to inactivity. Feel free to re-open if you still want these changes to be added.

**Pull request closed without merging**